File: TI2.4_EUREKABANK_BDDNUBE_REST_DOTNET_GR01/04. CLIESC/eurekabank_escritorio/src/ec/edu/gr03/controller/Cuenta.java
package ec.edu.gr03.controller;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Cuenta {
    @JsonProperty("NumeroCuenta")
    private String numeroCuenta;
    
    @JsonProperty("NombreCliente")
    private String nombreCliente;
    
    @JsonProperty("Saldo")
    private double saldo;
    
    @JsonProperty("Moneda")
    private String moneda;
    
    @JsonProperty("Estado")
    private String estado;

    public Cuenta() {
    }

    public String getNumeroCuenta() {
        return numeroCuenta;
    }

    public void setNumeroCuenta(String numeroCuenta) {
        this.numeroCuenta = numeroCuenta;
    }

    public String getNombreCliente() {
        return nombreCliente;
    }

    public void setNombreCliente(String nombreCliente) {
        this.nombreCliente = nombreCliente;
    }

    public double getSaldo() {
        return saldo;
    }

    public void setSaldo(double saldo) {
        this.saldo = saldo;
    }

    public String getMoneda() {
        return moneda;
    }

    public void setMoneda(String moneda) {
        this.moneda = moneda;
    }

    public String getEstado() {
        return estado;
    }

    public void setEstado(String estado) {
        this.estado = estado;
    }
}

File: TI2.4_EUREKABANK_BDDNUBE_REST_DOTNET_GR01/04. CLIMOV/app/src/main/java/ec/edu/espe/rest_dotnet_bank/data/models/Cuenta.kt
package ec.edu.espe.rest_dotnet_bank.data.models

import com.google.gson.annotations.SerializedName

data class Cuenta(
    @SerializedName("NumeroCuenta")
    val numeroCuenta: String = "",
    
    @SerializedName("NombreCliente")
    val nombreCliente: String = "",
    
    @SerializedName("Saldo")
    val saldo: Double = 0.0,
    
    @SerializedName("Moneda")
    val moneda: String = "",
    
    @SerializedName("Estado")
    val estado: String = ""
)
